app/components/support_interface/reference_with_feedback_component.rb

```diff
@@ -107,30 +108,36 @@ def name_row
       {
         key: 'Name',
         value: name,
+        action: 'name',
+        change_path: support_interface_application_form_edit_reference_details_path(reference.application_form, reference),
       }
     end
 
     def email_address_row
       {
         key: 'Email address',
         value: email_address,
+        action: 'email_address',
+        change_path: support_interface_application_form_edit_reference_details_path(reference.application_form, reference),
       }
     end
 
     def relationship_row
       {
         key: 'Relationship to candidate',
         value: relationship,
+        action: 'relationship',
+        change_path: support_interface_application_form_edit_reference_details_path(reference.application_form, reference),
       }
     end
 
     def feedback_row
-      if feedback
```

Comment:
I have removed this condition since a support user might want to add feedback as well as edit existing feedback.